_articles/download-data.md

```diff
@@ -0,0 +1,96 @@
+
+## Download Files
+
+### Command Line
+
+The Synapse command line client is *recommended* for downloading data from the portal as all data and file annotations can be downloaded with a single command.
+
+The command line client is installed with the Synapse Python client, therefore [Python 3](https://www.python.org/downloads/) is required to [install the Synapse command line client](http://python-docs.synapse.org/build/html/index.html#installation). [Login](https://python-docs.synapse.org/build/html/CommandLineClient.html#login) to Synapse.
+
+```
+synapse login
+```
+
+From Explore Data in the portal, select the **Download Options** icon and **Programmatic Options** to visualize the command to download the data subset.
+
+<img style="width: 25%;" src="/assets/images/programmatic-options-viz.png" alt="alt text">
+
+The command [`synapse get`](https://python-docs.synapse.org/build/html/CommandLineClient.html#get) with the `-q` argument downloads files from the entirety of the portal data that meet the specified condition. In this example, all *processed* and *metadata* files from the *MC-CAA* study will be downloaded. Execute the following command from the directory where you would like to store the files.
+
+```
+synapse get -q "SELECT * FROM syn11346063 WHERE ( ( "study" = 'MC-CAA' ) AND ( "dataSubtype" = 'processed' OR "dataSubtype" = 'metadata' ) )" 
+```
+
+Also in your working directory, you will find a *SYNAPSE_TABLE_QUERY_###.csv* file that lists the annotations associated with each downloaded file. Here, you might find helpful experimental details relevant to how the data was processed. Additionally, you will find important details about the file itself including the file version number.
+
+### R
+
+In order to download data programmatically with R, you need a list of synIDs that correspond to the files. For downloading a large set of files, we recommend using the Synapse Python client. The Python client has been optimized for multi-threaded download and will provide you with faster download speeds.
+
+Once you have identified the files you want to download from Explore Data, **Export Table** from Download Options.
+
+<img style="width: 25%;" src="/assets/images/export-table-viz.png" alt="alt text">
+
+You may choose to download the file as a *.csv* or *.tsv*. Files are named *Job-####*, where # includes a long set of numbers. Move this file to your working directory to proceed with the following steps. 
```

Comment:
would it be worth having code showing users that they could also get the exact query on download icon -> programmatic options, and import the table using `synTableQuery("...")`? Again, thinking here about having the full data download encoded in the script that users are writing so they are recording their work by default, rather than trying to suggest UI-based downloads.

Reply:
I think the fix here is add this to the UI. It is muddy to describe here (it would be something along the lines of copy and paste this partial command from programmatic options). I opened https://sagebionetworks.jira.com/browse/PORTALS-1571. 

Great point though, let's work towards that!